src/_P103_Atlas_EZO_pH_ORP_EC.ino

```diff
@@ -532,96 +566,117 @@ boolean Plugin_103(byte function, struct EventStruct *event, String& string)
 // The other containing an allocatted char array for answer
 // Returns true on success, false otherwise
 
-bool _P103_send_I2C_command(uint8_t I2Caddress, const String& cmd, char *sensordata) {
+bool _P103_send_I2C_command(uint8_t I2Caddress, const String &cmd, char *sensordata)
+{
+  sensordata[0] = '\0';
+
   uint16_t sensor_bytes_received = 0;
 
   byte error;
   byte i2c_response_code = 0;
-  byte in_char           = 0;
+  byte in_char = 0;
 
   String log = F("> cmd = ");
-          log += String(cmd);
-          addLog(LOG_LEVEL_DEBUG, log);
+  log += String(cmd);
```

Comment:
`cmd` is already a `String` object, so you can append it directly. No need to wrap it in a `String` constructor

Reply:
fixed